tests/thread_interrupt/enc/enc.c

```diff
@@ -0,0 +1,200 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <openenclave/corelibc/string.h>
+#include <openenclave/enclave.h>
+#include <openenclave/internal/print.h>
+#include <openenclave/internal/raise.h>
+#include <openenclave/internal/sgx/td.h>
```

Comment:
Since this test is SGX specific, I'd recommend moving these files down a level under tests/sgx/